frontend/src/components/InfoWindow/InfoWindow.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * The react component for the information window for each node
+ * in the concept tree.
+ */
+
+// handle right clicks with
+// https://stackoverflow.com/questions/31110184/react-synthetic-event-distinguish-left-and-right-click-events
```

Comment:
Should we move this into the ticket description instead?

Reply:
Why do we even have this link?

Reply:
@sultansidhu

Reply:
@rafiiary This is just to keep it in front of our eyes when we develop for handling right clicks. Info for right clicks was kinda hard to find so i put the most helpful link on here. Should I move them to ticket description instead? might get lost there tho, but im okay either way

Reply:
@sultansidhu put it in the Road Map Goals under the links section please 
https://docs.google.com/document/d/1Ujy8Y-5NUPA0RghRK2dCsw5JEuLjSkR0J8mer6uI5GA/edit?usp=sharing